frontend/public/locales/en/search.json

```diff
@@ -24,9 +24,11 @@
     "search.modal.save.action.cancel": "Cancel",
     "search.modal.save.action.save": "Save",
     "search.modal.save.input.description": "Description (120 character limit)",
+    "search.modal.save.input.description.placeholder": "Enter description for saved search",
     "search.modal.save.input.error.desc": "Enter search text",
     "search.modal.save.input.error.title": "Error",
     "search.modal.save.input.name": "Search name (50 character limit)",
+    "search.modal.save.input.name.placeholder": "Enter name for saved search",
```

Comment:
@dockerymick Can you review these 2 strings?
I wonder if this sounds better:
> Enter a name/description for this search query

Reply:
I agree. That sounds better, compared to what I have written.

Reply:
@dislbenn We could also keep it simple like "Enter a name" and "Enter a description (optional)"

@dockerymick Here are the strings in the UI for context.
![image](https://user-images.githubusercontent.com/4671325/118181534-dec1a180-b405-11eb-8252-a6313b9f6cc1.png)

Reply:
For line 27, I suggest: 

Enter a description for the saved search

OR 

Enter a description for your saved query.

For line 31 

Is the user required to enter **both** or _either one?_

I would suggest: 

Enter a name or description for this search query.

OR 

Enter a name and description for this search query.

Reply:
@dockerymick do you think we should add `(optional)` at the end of the description string?

Reply:
I would add it before the statement, so it would read as the following statement:

Optional: Enter a description for this search query

Reply:
Alright, it is fixed. Thanks @jlpadilla @dockerymick!